server/config/config.go

```diff
@@ -29,6 +35,9 @@ const (
 	// DefaultGRPCMaxSendMsgSize defines the default gRPC max message size in
 	// bytes the server can send.
 	DefaultGRPCMaxSendMsgSize = math.MaxInt32
+
+	// DefaultABCIClientType defines the default ABCI client type to use with cometbft.
+	DefaultABCIClientType = AbciClientTypeCommitting // [AGORIC]
```

Comment:
There's some more references to `DefaultABCIClientType` and `AbciClientType...` in the `server` package from the `Agoric` branch.  Please either leave some TODO comments as below, or complete the implementation.

```suggestion
	DefaultABCIClientType = AbciClientTypeCommitting // [AGORIC] TODO: Use in `DefaultConfig().BaseConfig.ABCIClientType` initialization below
	// [AGORIC] TODO: implement `FlagAbciClientType` in `server/start.go` module, `addStartNodeFlags`
```